google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -230,6 +242,7 @@ public static EnhancedBigtableStub create(EnhancedBigtableStubSettings settings)
     return mutateRowsCallable;
   }
 
+  /** Returns the callable chain created in {@link #createCheckAndMutateRowCallable()}. */
```

Comment:
nit:
This comment is probably less useful. The reason is that createCheckAndMutateRowCallable() is private and it is called in the constructor to get checkAndMutateRowCallable initialized. By pointing user directly to createCheckAndMutateRowCallable() creates confusion. I would simple just drop that or add "created in the constructor by invoking createCheckAndMutateRowCallable()".

Reply:
updated the doc